Server/Hash/hash_function.py

```diff
@@ -0,0 +1,48 @@
+
+class Hash:
+
+	#function to return hash of a string 
+	def hash_function(password):
+		final_hash = ""                                    # will contain the final hash created
+
+		ord_encryption = 0
+		for i in range(len(password)):
+			ord_encryption += ord(password[i])
+			final_hash += str(ord(password[i])%len(password))
+
+		value_encrypting = 0
+		get_encrypted_value_string = ""
+		for i in range(1,len(password)):
+			value_encrypting += max(ord(password[i]),ord(password[i-1])) - min(ord(password[i]),ord(password[i-1]))
+			temp = max(ord(password[i]),ord(password[i-1])) - min(ord(password[i]),ord(password[i-1]))
+			value_temp = max(ord(password[i]),ord(password[i-1])) - min(ord(password[i]),ord(password[i-1]))
+			if(value_temp > 32 and value_temp < 128):
```

Comment:
what is the purpose of this line? does it ignore other characters?

Reply:
this is because from ASCII value 1 to 32 character is blank so to ignore blank characters this line is used

Reply:
someone may have a space in their password, that is ASCII 32

Reply:
That will not have any effect, this is only to avoid space in generated hash.